buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -42,7 +42,7 @@
     private static final int DEFAULT_NUM_DIRECT_ARENA;
 
     private static final int DEFAULT_PAGE_SIZE;
-    private static final int DEFAULT_MAX_ORDER; // 8192 << 11 = 16 MiB per chunk
+    private static final int DEFAULT_MAX_ORDER; // 8192 << 9 = 4 MiB per chunk
```

Comment:
@chrisvest is this also something we want to consider for 4.1 ?

Reply:
I had a feeling we had discussed this before. Apparently I promised to look at this at some point: https://github.com/netty/netty/issues/8536

So yeah, let's back port this. 😅